src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -11,6 +11,14 @@
 
 
 class AzureKubernetesServiceScenarioTest(ScenarioTest):
+    @AllowLargeResponse()
+    def test_get_version(self):
```

Comment:
@gtxistxgao  hopefully this catches the get-versions break we had this week till we move it into managed cluster or some other api we keep up to date.